### ISSUE TITLE
Add account manager

### ISSUE THREAD

**Issue by JonathanMM** [andresth/Kandroid#32]
I use a kanboard on many organisation and I wish the possibility to use kandroid with many instance of kanboard (or have an easy way to switch quickly between instance)

**Comment by jpellegrini** [comment]
I agree that it would be great. I can easily install several instances of Kanboard; it would be good to be able to pick an instance from the app -- It would allow me to isolate contexts.
